osbuild/pipeline.py

```diff
@@ -94,7 +97,8 @@ def add_mount(self, name, info, device, target, options):
     def prepare_arguments(self, args, location):
         args["options"] = self.options
         args["meta"] = {
-            "id": self.id
+            "id": self.id,
+            "source-epoch": self.source_epoch
```

Comment:
A `args["build"]` could also make sense, where we communicated all build related stuff. but I think meta is also fine, so let's not complicate things now.